independent-publisher-connectors/PowerBI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1062 @@
+                        "description": "id"
+                      },
+                      "isReadOnly": {
+                        "type": "boolean",
+                        "description": "isReadOnly"
+                      },
+                      "isOnDedicatedCapacity": {
+                        "type": "boolean",
+                        "description": "isOnDedicatedCapacity"
+                      },
+                      "capacityMigrationStatus": {
+                        "type": "string",
+                        "description": "capacityMigrationStatus",
+                        "title": "Capacity Migration Status"
+                      },
+                      "type": {
+                        "type": "string",
+                        "description": "type"
+                      },
+                      "state": {
+                        "type": "string",
+                        "description": "state"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "name"
+                      }
+                    }
+                  },
+                  "description": "value"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Groups as Admin",
+        "description": "Returns a list of workspaces for the organization",
+        "operationId": "GroupsGetGroupsAsAdmin",
+        "parameters": [
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+			"x-ms-summary" : "Use the expand property to expand certain properties in the group return object.",
+            "description": "Use the expand property to expand certain properties in the group return object."
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+			"x-ms-summary" : "Enter filter criteria (eg type eq 'Workspace')",
+            "description": "Use the filter parameter to filter groups on a property. In the default example this will contain a filter to get only groups of type 'Workspace'."
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+			"x-ms-summary" : "Number of records to return",
+            "description": "Use the top parameter to set the top n results",
+            "default": 500,
+            "format": "int32"
+          },
+          {
+            "name": "$skip",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+			"x-ms-summary" : "Number of records to skip",
+            "description": "Use the top parameter to skip the first x amount of groups that are being returned."
+          }
+        ]
+      }
+    },
+    "/v1.0/myorg/admin/groups/{groupId}/users": {},

```

Comment:
@paulculmsee Please remove all the empty operations as they will not add any value to the connector.

Reply:
done